### ISSUE TITLE
delete container when $timeout has occurred" doesn't work well when the name is "dynamic".

### ISSUE THREAD

**Issue by fishzle** [stoically/temporary-containers#402]
I have been using "%domain% #" as per #140 as the container name for about 2 weeks now. Thank you - very useful.

Though I didn't take all the advice in that post. I didn't make changes to mousebindings - I mostly use LMB to open tabs, and occasionally ctrl-LMB. (RMB is faulty)

I'm seeing previous containers (hundres of them) that were tmp% still visible in the MC menu.
Also new closed windows that are %domain%# are also still visible, after the timeout. Also tens of these from the last couple weeks.

So, in summary, looks like container aren't being deleted after $timeout with the old name scheme or when the name is "dynamic".

It's possible it's a by-product of my plugins / settings. But before I spend a couple of days recreating my profile, re-installing addons, and re-configuring everything - does anyone else see this behaviour?

**Comment by stoically** [comment]
Internally TC works based on the container id, not the name - so I don't think that's the cause. Maybe you're hitting #371?

**Comment by fishzle** [comment]
Spot on. I'll close this.

**Comment by crssi** [comment]
@fishzle 
Was it MAC?

**Comment by fishzle** [comment]
I read the notes in #371.

I am using MAC.
Found that I had 19GB in weave logs.
I disabled sync of add-ons.
deleted all 19GB of logs.
I backed up my profile.
I deleted all the old ones and kept a few permanent ones with the snippet in https://github.com/stoically/temporary-containers/issues/371#issuecomment-617626349

left it overnight (I needed to sleep) and sync.

This morning the old containers are disappearing as expected. MAC has been active the whole time.

So, a guess is that the 200kb cap was one part of the failure.

I have just re-enabled add-on sync (with both MAC and TC in use) and will check it again later.

**Comment by fishzle** [comment]
After 7h of active use of firefox 75.0 (64bit) with MAC and TC.
Old containers are disappearing as expected.
The issue hasn't re-appeared on my primary desktop.

However, I've just started up my laptop (hasn't been on for a few weeks).
It has a different set of old containers.
add-on sync is on.
I've forced a firefox sync.

But the old containers didn't go. So, I manually removed them.

They didn't show up on my primary desktop.

**Comment by crssi** [comment]
I had a lot of problem with MAC in the past, so I have replaced it with Containerise... and its also easier to copy configuration to another computer.
